man/systemd.network.xml

```diff
@@ -749,6 +749,14 @@
           </listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>Scope=</varname></term>
+          <listitem>
+            <para>The scope of the address, which can be <literal>global</literal>,
+            <literal>link</literal> or <literal>host</literal> or a unsigned integer. Defaults to
```

Comment:
"a unsigned" → "an unsigned"

Also, please document the valid range here. Is "0" permitted? and what is the largest value accepted?